src/flow-spec/IBlock.ts

```diff
@@ -339,7 +339,7 @@ export function setContactProperty(block: IBlock, context: IContext): void {
 }
 
 function setSingleContactProperty(property: SetContactProperty, context: IContext): void {
-  const value = evaluateToString(property.property_value, createEvalContextFrom(context))
+  const value = EvaluatorFactory.create().evaluate(property.property_value, createEvalContextFrom(context))
```

Comment:
(I'm switching out of using `evaluateToString` here because it uses `wrapInExprSyntaxWhenAbsent` - see line 327)